apps/web/vibes/soul/primitives/inline-email-form/index.tsx

```diff
@@ -74,8 +68,13 @@ export function InlineEmailForm({
         </div>
       </div>
       {errors.map((error, index) => (
-        <FieldError key={index}>{error}</FieldError>
+        <FormStatus key={index} type="error">
+          {error}
+        </FormStatus>
       ))}
+      {form.status === 'success' && lastResult?.successMessage != null && (
+        <FormStatus>{lastResult.successMessage}</FormStatus>
+      )}
```

Comment:
Maybe for this form we do a toaster if we can't figure out a way to avoid the layout shift. We could also replace the form itself with the success message and match the height

Reply:
Would you be okay if we track this design issue in a different issue?

Reply:
Yeah, let's do that. Can we please make a Linear issue so we don't lose track of it?